src/mkindex_algo.hpp

```diff
@@ -69,6 +68,7 @@ auto loadSubjSeqsAndIds(LambdaIndexerOptions const & options)
     // see http://www.uniprot.org/help/accession_numbers
     // https://www.ncbi.nlm.nih.gov/Sequin/acc.html
     // https://www.ncbi.nlm.nih.gov/refseq/about/
+    //TODONE: I am not sure whether we should spent any time on the taxonomy TODOs...
```

Comment:
I think we should just fix the taxonomy in general including testing however that will work and then call it a day?